pyseir/inference/model_fitter.py

```diff
@@ -225,18 +227,18 @@ def set_inference_parameters(self):
             "t_break",
             "eps2",
             "t_delta_phases",
-            "test_fraction",
-            "hosp_fraction",
+            # "test_fraction",
+            # "hosp_fraction",
             "log10_I_initial",
         ]
         if self.fips in overwrite_params_df["fips"].values:
             this_fips_df = overwrite_params_df.loc[overwrite_params_df["fips"] == self.fips]
             for param in INITIAL_PARAM_SETS:
                 self.fit_params[param] = this_fips_df[param]
 
-        self.fit_params["fix_hosp_fraction"] = self.hospitalizations is None
-        if self.fit_params["fix_hosp_fraction"]:
-            self.fit_params["hosp_fraction"] = 1
+        # self.fit_params["fix_hosp_fraction"] = self.hospitalizations is None
```

Comment:
Can you add some context as to why this is commented out in a comment? Generally I find it helpful to comment why it's commented (as opposed to removed), are you planning on brining this code back ?